### PR TITLE
parkinsons_model2

### PR DESCRIPTION
**Pull request by Aravind210202**
## What are the changes?
This file consists of more models necessary for the prediction of Parkinsons Disease in a person .The models being used in this are the KNN,Gradient Boost,XGboost,MLP and the Naive Bayes .

## Who worked on the changes?
Aravind Srivatsan

## New Components/Features
The new features added were the models which are:
KNN:This is used due to it being a non-parametric algorithm and is very effective in classification
Gradient Boost:It is an ensemble learning technique and it focuses on minimising bias between the data and the model predictions.It is also robust to overfitting
XGBoost:Used for its efficiency and works well with large data sets
MLP:Ability to Model Complex, Non-linear Relationships and does good feature extraction
Naive Bayes:Probabilistic classification and can handle high dimensional datasets

## Enhanced Components/Features
Some of the models provide an increased accuracy compared to the ones which already exist 

## Other Changes/Fixes
The visualisation is being done in Bar plots as they will help in seeing the results in a different manner 

## Additional Notes
Unfortunately the Naive Bayes provides a significantly lower accuracy compared to the the other models due to the distribution of features 

## Checklist
- [x ] Existing, relevant documentation has been updated to reflect my changes 

## Screenshots or example output

Screenshot 2024-09-23 at 5 51 18 PM

### CODE REVIEW

**Comment by Manan2711** [comment]
There are no major issues with the file and can be merged. Thank you!
